Assets/Huawei/Demos/ML/LanguageDetection/LanguageDetectionDemoManager.cs

```diff
@@ -0,0 +1,188 @@
+using Assets.Huawei.Scripts.ML.LangDetector;
+using HuaweiMobileServices.ML.LanguageDetection;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class LanguageDetectionDemoManager : MonoBehaviour
+{
+
+    private const string Tag = "[HMS] LanguageDetectDemoManager ";
+    public Button onlineButton;
```

Comment:
Can you edit all fields in the format shown below?

`[SerializeField] private Button m_onlineButton`

[SerializeField] provides to shown field in Inspector.